Makefile

```diff
@@ -0,0 +1,29 @@
+.PHONY: all integration-tests deployer geth-l2 batch-submitter data-transport-layer test
+
+all:
+	./build-local.sh
+
+up-local:
+	./up.sh -l
+
+integration-tests:
+	./build-local.sh -s integration_tests
+
+deployer:
+	./build-local.sh -s deployer
+
+geth-l2:
+	./build-local.sh -s geth_l2
+
+batch-submitter:
+	./build-local.sh -s batch_submitter
+
+data-transport-layer:
+	./build-local.sh -s data_transport_layer
+
+# For developing against published docker images
+up: 
+	./up.sh
+
+test:
```

Comment:
This didn't work for me when I ran:

```bash
$ make test -p x-domain
```

Reply:
It ran all of the integration tests instead of just running the single test suite. I noticed that it did not pass through the argument